### PR TITLE
use esmd package

### PR DESCRIPTION
**Pull request by jimisaacs**
This is just a first step to make sure the code is separated from the repo.

I know it's a long road to separate things more appropriately from this. Though wanted to at least see if you thought this was an ok starting point. All I've basically done is: moved code, separated the embed files from build specific to frontend specific, and created the config interface. That's all.

The logic to know how to serve the frontend is still in esmd, but again, I realize there's a long way to go here.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,18 +1,46 @@
-module esm.sh
+module github.com/esm-dev/esm.sh
 
-go 1.16
+go 1.18
```
[the remark this answers is not in the document]
Comment (ije) [reply go.mod:3]: nice!

**Review by ije — APPROVED** [review]
LGTM, nice work!

**Comment by ije** [comment]
please merge the master branch, the testing was failed by type check in deno,  i will look into it~

**Comment by artempanko** [comment]
hi, I would like to start contributing to this project, any plans for merging this pr?
maybe I can take it
@ije

**Comment by jimisaacs** [comment]
Please do! Things in life have taken over my availability.

**Comment by artempanko** [comment]
but could you please explain why are you moving all go server code to a separate repo?

**Comment by jimisaacs** [comment]
Was just a starting point. The plan was to start somewhere with move much of golang in the other repo, then carve away the parts that are server specific, bring them back to esm.sh codebase, and eventually make esmd into the original vision of a separation of concerns that allows for building with "lego" blocks versus depending on "server flags for all batteries included" kind of a setup.

**Comment by ije** [comment]
@jimisaacs @artempanko i am going to close this. i think separate repo will be very painful for dev.

**Pull request closed without merging**